scripts/common.ts

```diff
@@ -15,6 +15,7 @@ export async function deployAndVerify(
   await new Promise((resolve) =>
     setTimeout(() => resolve(), 30000)
   );
+  /**
```

Comment:
If this is breaking w/o the etherscan API key, can we just make it optional and only run the `exec` if the ENV var is present?

Reply:
This is breaking with the API key, this is the reason it's giving me: **Failed to verify contract. Reason: Max rate limit reached, please use API Key for higher rate limit**

Reply:
Ah got it. Okay let's just leave this commented for now :P